src/webgpu/shader/execution/shader_io/fragment_builtins.spec.ts

```diff
@@ -287,7 +287,7 @@ function isTriangleClockwise(windowPoints: readonly number[][]) {
     const p1 = windowPoints[(i + 1) % 3];
     sum += p0[kX] * p1[kY] - p1[kX] * p0[kY];
   }
-  return sum < 0;
+  return sum >= 0;
```

Comment:
https://math.stackexchange.com/questions/1324179/how-to-tell-if-3-connected-points-are-connected-clockwise-or-counter-clockwise

Seems that clockwise is <0 though. Maybe there is something else that's flipped?

Reply:
IIRC it's that in window coords  0,0 is at the top left in WebGPU vs OpenGL where it's bottom left, so triangles are flipped.

Reply:
Got it.